packages/demo/src/components/App/index.js

```diff
@@ -14,13 +14,21 @@ const TaglineRe = /^(.*)\[Cerebral\](.*)$/
 export default connect(
   {
     t: translations,
-    loggedIn: 'user.$loggedIn'
+    loggedIn: 'user.$loggedIn',
+    $loading: 'app.$loading',
```

Comment:
PS: I tried to keep props and signals sorted (but clearly missed some). I can fix this later if you want.